pkg/config/setup/security_agent.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package setup
+
+import (
+	"net"
+	"strconv"
+
+	pkgconfigmodel "github.com/DataDog/datadog-agent/pkg/config/model"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// DefaultSecurityAgentCmdPort is the default port used by security-agent to run a runtime settings server
+const DefaultSecurityAgentCmdPort = 5010
```

Comment:
it seems to be used only in `func GetSecurityAgentAPIAddressPort` shouldn't be better to avoid the usage of global var and declare it in the scope of the function ?

Reply:
It's also used in pkg/config/setup/config.go
```
config.BindEnvAndSetDefault("security_agent.cmd_port", DefaultSecurityAgentCmdPort)
```

Reply:
okay my bad did not saw it ;)